config.json

```diff
@@ -391,6 +392,7 @@
         "productgallery_size": "500x659",
         "product_size": "500x659",
         "productthumb_size": "100x100",
+        "productview_thumb_size": "50x50",
```

Comment:
Not needed. This only needs to be added once to the global settings at line 82

Reply:
Ah, I was wondering about that actually, but the other sizes had the same duplication going on. Should I clean up some of the other image sizes that are duplicates as well?

Reply:
Yes please! That will be awesome :)
Just make sure to remove identical fields/values

Reply:
Alright, I went wild and deleted a whole bunch of duplicates. I didn't touch any of the color settings (even when they were technically duplicates) because they struck me as sort of coincidentally duplicated.